src/App.tsx

```diff
@@ -1,155 +1,78 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { AddTodo } from './сomponents/AddTodo';
+import { TodosFilter } from './сomponents/TodosFilter';
+import { TodoList } from './сomponents/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Filter } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 11337;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterStatus, setFilterStatus] = useState<Filter>(Filter.all);
+  let filteredTodos: Todo[];
```

Comment:
Avoid using 'let' for variable declaration. Instead, you can use the useState hook to manage the state of the filteredTodos and update it when filterStatus changes.